dev_configs/service-portal_linux.js

```diff
@@ -0,0 +1,9 @@
+// This is a sample config file for the following scenario
+// You have the service portal running locally on your dev machine on port 5000
+// The Insights UI is running on CI dev cluster
+module.exports = {
```

Comment:
@mkanoor Only change I can see.  Since these are `js` files.  We should move them into `app/assets/javascripts/<file_name>.js` so they follow the rails standard file structure for js files.

Reply:
@syncrou These are only for developers who want to run their service locally but access the UI from the dev cluster. They are not part of the product its purely for developers. The insights proxy uses javascript for configuration but if they changed it to a yam file it would still be doing configuration. So I am not sure if these have to to become a java script assets its better if they are in an isolated directory for developers to use.

Reply:
@mkanoor, yeah, that all makes sense, and I wasn't misunderstanding the use case.  The directory name and location is a challenge. I suggested `app/assets` because it is common place to put any js files dev or otherwise.

Reply:
Was discussion with @Fryguy and thought `config/spandx` would be a better choice as these are not js files for the application, and it avoids creating a new top-level directory.